hoomd/md/TwoStepNVTMTK.cc

```diff
@@ -451,7 +451,10 @@ void TwoStepNVTMTK::thermalizeThermostatDOF(unsigned int seed, unsigned int time
     Scalar sigmasq_t = Scalar(1.0)/((Scalar) g*m_tau*m_tau);
 
     bool master = m_exec_conf->getRank() == 0;
-    hoomd::RandomGenerator rng(hoomd::RNGIdentifier::TwoStepNVTMTK, seed, timestep);
+    hoomd::RandomGenerator rng(hoomd::Seed(hoomd::RNGIdentifier::TwoStepNVTMTK,
+                                           timestep,
+                                           m_sysdef->getSeed()),
+                                      hoomd::Counter());
```

Comment:
In this case, I think I will solve it by including one of the particle tags from the group since those are disjoint. I suppose a static counter could be used in other cases if there were no other option. I can't think of any cleaner general solution.

Reply:
done for NVT and NPT